src/Http/Controllers/CartItemController.php

```diff
@@ -16,11 +16,21 @@ public function store(Request $request)
             'product_type' => 'required|string',
             'product_id' => 'required',
             'quantity' => 'nullable|numeric',
-            'options' => 'nullable|array'
+            'options' => 'nullable|array',
+            'multi' => 'nullable|string'
         ]);
 
         $product = $request->product_type::findOrFail($request->product_id);
-        Cart::add($product, $request->quantity ?? 1, $request->options);
+
+        if ($request->quantity > 1 &&  $request->multi) {
+            $multiOptions = $request->options;
+            foreach ($request->options[$request->multi] as  $value) {
+                $multiOptions[$request->multi] = $value;
+                Cart::add($product, 1, $multiOptions);
+            }
```

Comment:
I think it should be something like this:
```php
foreach ($request->options as $options) {
    Cart::add($product, 1, $options);
}
```

Reply:
i don't think it's work.
did you tryed this and it worked?